### ISSUE TITLE
need new action in EML OPTION EXPORT WITH SUB/SUB/...  FOLDERS

### ISSUE THREAD

**Issue by karadevnet** [thundernest/import-export-tools-ng#229]
**hi, i am Kaloyan Radev and i download and install this super addon for thunderbird. but see the options in export in EML format have not a many to choose. IN OPTION EXPOSRT IN EML FORMAT need to add extra option to export and/or other sub/sub/sub/.... folder also in EML format each file in his own sub/sub/sub/.... folder to make in HDD same structure with EML files and sub/sub/sub/.... folder like in real email hosting.

i try some other options but addon just create some SMF files and that is all. i need a SELECT a post box / inbox of my email and when i choose EXPORT IN EML WITH FOLDERS(structure) i HDD addon module to export same structure of selected inbox mail like a real one. i hope i was understand what is need to do. i know in not hard, just need to add corect software for this option in export function

i am also a programmer for hardware pic microcontrollers and i know well that is not so hard when you already have a base functions of software.

for this now EXPORT IN EML make this just for selected folder, need to make for all email inbox sub/sub/sub/.... folders with same structer like original in server
best regards i hope this will be ready soon, my inbox mail is near by 8gb and i need to export in hdd to have a copy backup. also can add option to export WITH or WITHOUT attachment for all EML in sub/sub/sub/.... folders like other export options have this**

**Comment by cleidigh** [comment]
@karadevnet 
you are correct,  this is important missing feature.  it is on a long list of needed enhancements.
it is not trivial, I am not sure when I will have the time for it yet.
Christopher
